### PR TITLE
'analyse' is a valid word in British English

### PR DESCRIPTION
**Pull request by JMcKiern**
British English tends to use analyse in stead of analyze.

See https://writingexplained.org/analyse-or-analyze-difference

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -1244,18 +1244,17 @@ analitic->analytic
...
-analize->analyze
-analized->analyzed
-analizer->analyzer
-analizes->analyzes
-analizing->analyzing
+analize->analyze, analyse,
+analized->analyzed, analysed,
+analizer->analyzer, analyser,
+analizes->analyzes, analyses,
+analizing->analyzing, analysing,
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt]: If it's misspelled with the letter of said dialect then we only correct it for said dialect.

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -1244,18 +1244,17 @@ analitic->analytic
...
 analogeous->analogous
 analoguous->analogous
 analoguously->analogously
 analyis->analysis
-analysator->analyzer
-analysers->analyzers
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt]: Why was this omitted?

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -1244,18 +1244,17 @@ analitic->analytic
...
-analysies->analyzes
+analysator->analyzer, analyser,
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt]: Based on what I wrote above, please remove `analyzer`

File: codespell_lib/data/dictionary.txt [hunk 4]
```diff
@@ -1244,18 +1244,17 @@ analitic->analytic
...
+analysies->analyzes, analyses,
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt]: Suggested: `analysies->analyses, analysis,`

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (JMcKiern) [reply codespell_lib/data/dictionary.txt]: Oh yeah, sorry. That's obvious. I'll change it now.

File: codespell_lib/data/dictionary.txt [hunk 5]
```diff
@@ -1239,27 +1239,32 @@ amung->among
...
-analysies->analyzes, analyses,
+analysator->analyser
+analysies->analyses, analysis,
 analysus->analysis
 analysy->analysis
 analyticall->analytical, analytically,
 analyticaly->analytically
 analyticly->analytically
+analyzator->analyzer
+analyzies->analyzes
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt]: Suggest: `analyzies->analysis, analyses, analyzes`

File: codespell_lib/data/dictionary.txt [hunk 6]
```diff
@@ -1239,12 +1239,15 @@ amung->among
...
+analised->analysed
+analiser->analyser
 analises->analysis, analyses,
+analising->analysing
 analitic->analytic
 analitical->analytical
 analitically->analytically
 analiticaly->analytically
-analize->analyze
+analize->analyze, analyse,
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt]: Why did you add `analyse` ? :man_facepalming:

Comment (JMcKiern) [reply codespell_lib/data/dictionary.txt]: Woops, sorry. Forgot to remove that one

**Comment by luzpaz** [comment]
Please squash down you commits in to one commit. thanks

File: codespell_lib/data/dictionary.txt (hunk 4 above)
Comment (peternewman) [reply codespell_lib/data/dictionary.txt]: I'd agree with @JMcKiern 's suggestion of the zes variant too, resolving.
